public/style.css

```diff
@@ -10,7 +10,11 @@
 .position{
   position: fixed;
 }
-
+.center_img{
```

Comment:
If you do a search for `.center` and `.db` [here](http://tachyons.io/docs/table-of-styles/) you can see its identical